### PR TITLE
Disable optimizer on Gnosis-Safe contracts

### LINKED ISSUE #7287: Fix external Gnosis test
Gnosis-Safe is currently failing and needs to be fixed: https://circleci.com/gh/ethereum/solidity/167286?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

### PR DESCRIPTION
**Pull request by erak**
Fixes https://github.com/ethereum/solidity/issues/7287.

### CODE REVIEW

File: test/externalTests/gnosis.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ function test_fn { npm test; }
 
 function gnosis_safe_test
 {
-    OPTIMIZER_LEVEL=1
+    OPTIMIZER_LEVELS=0
```
Comment (erak) [inline test/externalTests/gnosis.sh:33]: This is the main change.

File: test/externalTests/common.sh [hunk 2]
```diff
@@ -189,20 +189,30 @@ function run_test
 
     force_solc "$CONFIG" "$DIR" "$SOLJSON"
 
-    printLog "Checking optimizer level..."
-    if [ -z "$OPTIMIZER_LEVEL" ]; then
-        printError "Optimizer level not found. Please define OPTIMIZER_LEVEL=[1, 2, 3]"
+    printLog "Checking optimizer levels..."
+    if [ -z "$OPTIMIZER_LEVELS" ]; then
+        printError "Optimizer level not found. Please define OPTIMIZER_LEVELS=[0, 1, 2, 3, 4, 5]"
```
Comment (bshastry) [inline test/externalTests/common.sh:194]: old -> new (renaming)
1 -> 5
2 -> 3
3 -> 2

Newly added levels
0 -> legacy optimizer disabled
1 -> legacy optimizer enabled
4 -> legacy optimizer disabled, new optimizer enabled

Did I get that right?

**Review by bshastry — COMMENTED** [review]
Looks good to me. Main comment regarding the semantics of the newly added levels. Perhaps it would be helpful to add a block comment about what each level means in the external tests common bash script.

File: test/externalTests/common.sh (hunk 2 above)
Comment (erak) [reply test/externalTests/common.sh:194]: Yes, that's exactly what it means :)

**Comment by erak** [comment]
Closing this since disabling the optimizer is not needed anymore after https://github.com/gnosis/safe-contracts/pull/126 got merged.

**Pull request closed without merging**